### ISSUE TITLE
When creating a new project the default path should be the same as the parent of your last project

### ISSUE THREAD

**Issue by louisgjohnson** [godotengine/godot-proposals#1312]
**Describe the project you are working on:** N/A

**Describe the problem or limitation you are having in your project:** When creating a new project it doesn't suggest the last place I had a project e.g: `C:/Users/Louis/Desktop/Godot/`

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
When the user creates a new project is suggests putting that new project inside the folder of where you last had a project.

last: `C:/Users/Louis/Desktop/Godot/ProjectA`
suggested: `C:/Users/Louis/Desktop/Godot/ProjectB`

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**

1. Check if the user has a previous godot project
2. if so suggest that parent folder as the new location for your project with the project name as the folder container
3. else use the default path

**If this enhancement will not be used often, can it be worked around with a few lines of script?:** Not sure

**Is there a reason why this should be core and not an add-on in the asset library?:** It seems like a lot of programs and other game engines work this way and it's a lot more convenient

**Comment by KoBeWi** [comment]
There is a default project path in Editor Setttings. If setting it manually is a problem we could add a checkbox when creating project to set current path as default.

**Comment by louisgjohnson** [comment]
> Editor Setttings. If setting it manually is a problem we could add a checkbox when creating project to set current path as default.

Yeah I guess part of my problem is there's no way to do it from the editor when you're creating a new project

**Comment by Zireael07** [comment]
Ooo. I was sure it was already requested when I saw the discussion yesterday but somehow my search fu failed.

Can someone edit the proposal title to have the word 'folder' and/or 'remember'?

**Comment by YuriSizov** [comment]
> Can someone edit the proposal title to have the word 'folder' and/or 'remember'?

I've edited the OP instead, it should still work for searches. I couldn't find a way to fit "remember" in there without making it less technically correct.

**Comment by passivestar** [comment]
Never realized that I can change the default project path because it doesn't make sense that this setting wouldn't be accessible through the project manager itself, it's practically undiscoverable. I like this proposal because a lot of other software remembers the last path and it greatly improves usability without requiring users to manually change any editor setting

**Comment by KoBeWi** [comment]
Should the default path be remembered every time, only first time, or per request (e.g. via checkbox in the dialog)?

**Comment by akien-mga** [comment]
> Should the default path be remembered every time, only first time, or per request (e.g. via checkbox in the dialog)?

IMO it should be per request.

Remembering it every time means that this is not an editor setting anymore, but just some metadata.

The editor setting is meant to be something users can configure once for their main location for projects, and they can still put projects as needed somewhere else without overriding this. Changing this behavior would IMO break the purpose of that setting.

On the other hand adding a checkbox to allow overriding it on demand (which can default to enabled if the setting is empty, thus making the first configuration easier), makes sense to me.
